packages/yoga/src/Progress/native/Progress.jsx

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import styled from 'styled-components';
 import { string, number, shape, oneOfType, oneOf } from 'prop-types';
+import {
+  backgroundColor,
+  border,
```

Comment:
@mmartinsoliveira here we are importing only the `border` prop, but on the tests, we are testing the `borderRadius`.
Should it be `borders` to get all system props associated to the border or just be the `border` one?

Reply:
Solved, @ericcleao